sdk/sdk.go

```diff
@@ -19,10 +19,10 @@ type SDK struct {
 }
 
 // New creates a new SDK with given options.
-func New(c container.Container, serviceDB database.ServiceDB, instanceDB database.InstanceDB, execDB database.ExecutionDB) *SDK {
+func New(c container.Container, serviceDB database.ServiceDB, instanceDB database.InstanceDB, execDB database.ExecutionDB, endpoint string) *SDK {
```

Comment:
not blocking but this one start to be big, maybe we could create a struct like `SDKConfig` to pass all this

Reply:
Yes, but this will be changed with cosmossdk